mkdocs.yml

```diff
@@ -14,6 +14,11 @@ nav:
       - Best Practices: 'lf-tags/best-practices.md'
       - Example Usecase: 'lf-tags/example-usecase.md'
       - Limitations: 'lf-tags/limitations.md'
+  - 2. Cross Account Data Sharing: 'datasharing/cross-account/index.md'
```

Comment:
Please work with Aarthi to consolidate/reconcile the changes she made for data sharing. It seems like she made very similar changes.

Reply:
ack